src/components/ui/ScheduleItem/ScheduleItem.module.scss

```diff
@@ -0,0 +1,46 @@
+.item {
+  display: flex;
+  flex-direction: row;
+  padding: var(--space-m);
+  border-radius: var(--border-radius-default);
+}
+
+.iconContainer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  width: var(--icon-container-size);
+  height: var(--icon-container-size);
+  border-radius: var(--border-radius-default);
+  background-color: var(--color-white);
+  margin-right: var(--space-m);
+}
+
+.icon {
+  width: var(--icon-size);
+  height: var(--icon-size);
+}
+
+.textContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-size: var(--font-size-m);
+  color: var(--color-white);
+}
+
+.lightBlue {
```

Comment:
maybe we could nest the color related classes inside `.item` to couple them together and prevent from reusing these classes in other context?